run-make-check.sh

```diff
@@ -68,8 +68,8 @@ function run() {
     if test -x ./do_cmake.sh ; then
         $DRY_RUN ./do_cmake.sh || return 1
         cd build
-        $DRY_RUN make $BUILD_MAKEOPTS || return 1
-        $DRY_RUN ctest $CHECK_MAKEOPTS --output-on-failure || return 1
+        export CTEST_OUTPUT_ON_FAILURE=1
+        $DRY_RUN make $BUILD_MAKEOPTS check || return 1
```

Comment:
@tchaikov what was the reason for this switch? In `make check` BUILD_MAKEOPTS don't get transferred to ctest so the newer version would be slower than the older version.